### PR TITLE
New Errand Component Skeleton Complete

### PR DESCRIPTION
**Pull request by aaronfife**
## Description
- fixed the error on the dashboard component when viewing the web version
- added the nav bar component to the dashboard
- created the skeleton for the add errand component
- added the nav bar to the add errand component

## How to test
both should be available for viewing on iOS or web

## Notes
i noticed that the web view for the css does not always match the iOS view for the css. not sure which one is priority or if this even matters but would like to know if you guys are seeing the same thing

## Issue tracking
https://trello.com/c/3xW2NL11/39-add-errand-component-skeleton

## Current Image
Screen Shot 2022-01-03 at 2 41 46 PM

### CODE REVIEW

File: client/components/ErrandTracker/ErrandTrackerContainer.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import errandState from '../../state/atoms/errands';
 
 console.log(GOOGLE_MAPS_API_KEY)
 import TopBar from './TopBar/TopBar.jsx';
-import ErrandMap from './errandMap.jsx';
```
Comment (OjeikuA) [inline client/components/ErrandTracker/ErrandTrackerContainer.jsx:23]: I noticed that if this is made UpperCase, my app crashes. I always have to fix it back to camel case (see Louisa's update in Slack).

Comment (OjeikuA) [reply client/components/ErrandTracker/ErrandTrackerContainer.jsx:23]: I'll approve this all, but just figured I'd flag it for others' attention

**Review by OjeikuA — APPROVED** [review]
Looks great other than my one comment about ErrandMap vs. errandMap

File: client/components/ErrandTracker/ErrandTrackerContainer.jsx (hunk 1 above)
Comment (shabbyblue16) [reply client/components/ErrandTracker/ErrandTrackerContainer.jsx:23]: I know the filename is `errandMap` so I'd think the import file would have to be camelCase as well

**Review by shabbyblue16 — APPROVED** [review]
well done

**Pull request merged**